src/components/SettingsModal/index.jsx

```diff
@@ -149,16 +149,33 @@ export default class SettingsModal extends React.Component {
                   />
                 </Col>
               </Form.Group>
+              <Form.Group as={Row}>
+                <Form.Label column sm="8">
+                  Reset to Defaults
+                </Form.Label>
+                <Col sm="4">
+                  <Button
+                    variant="secondary"
+                    onClick={this.handleDefault}
+                    type="reset"
```

Comment:
Huh, great question. I think because it's in a <button> tag and doesn't connect to a form the `onClick` probably overrides any other behavior. I will switch it to `type="button"` to be safe. 

More interestingly Mozilla hates the `reset` button in general haha.
>The button resets all the controls to their initial values, like <input type="reset">. (This behavior tends to annoy users.)

>You should usually avoid including reset buttons in your forms. They're rarely useful, and are instead more likely to frustrate users who click them by mistake (often while trying to click the submit button).

Reply:
Switch to button type here: https://github.com/natcap/invest-workbench/pull/132/commits/bd3e1e8157243208d9d3f96b2d4721c701dc42e6